contents/docs/self-host/deploy/snippets/installing.mdx

```diff
@@ -5,3 +5,9 @@ helm repo add posthog https://posthog.github.io/charts-clickhouse/
 helm repo update
 helm install -f values.yaml --timeout 20m --create-namespace --namespace posthog posthog posthog/posthog
 ```
+
+<details>
+<summary>Troubleshooting</summary>
+
+If you don't see some pods come up (e.g. `chi-posthog-posthog-0-0-0` pod is missing after deleting a previous instance and trying to install PostHog again), try running `helm upgrade -f values.yaml -n posthog posthog posthog/posthog`.
```

Comment:
Why the clickhouse pod should be missing?

Reply:
no idea, it just happens often - haven't had time to figure out why yet.

Reply:
```suggestion
If you don't see some pods or services come up (e.g. `chi-posthog-posthog-0-0-0` pod or `clickhouse-posthog` service is missing), try running `helm upgrade -f values.yaml -n posthog posthog posthog/posthog`.
```